provider/datagen/tests/verify-zero-copy.rs

```diff
@@ -46,7 +46,16 @@ fn main() {
     let selected_locales = icu_testdata::locales();
 
     let converter = DatagenProvider {
-        source: SourceData::internal_offline_latest_tested_subset(),
+        source: SourceData::default()
```

Comment:
Nit: This is in `datagen` so you can use the API function

Reply:
Integration tests only get the public API